docker-compose.yaml

```diff
@@ -0,0 +1,204 @@
+    stop_grace_period: 1s
+
+  sequencer2:
+    image: ghcr.io/espressosystems/espresso-sequencer/sequencer:main
+    ports:
+      - "$ESPRESSO_SEQUENCER2_API_PORT:$ESPRESSO_SEQUENCER_API_PORT"
+    # Run the API server (with options taken from the environment)
+    command: sequencer -- http -- query
+    environment:
+      - ESPRESSO_SEQUENCER_ORCHESTRATOR_URL
+      - ESPRESSO_SEQUENCER_DA_SERVER_URL
+      - ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL
+      - ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_SEQUENCER_STORAGE_PATH
+      - ESPRESSO_SEQUENCER_L1_PROVIDER
+      - ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG
+      - RUST_LOG
+      - RUST_LOG_FORMAT
+    depends_on:
+      orchestrator:
+        condition: service_healthy
+    stop_grace_period: 1s
+
+  sequencer3:
+    image: ghcr.io/espressosystems/espresso-sequencer/sequencer:main
+    ports:
+      - "$ESPRESSO_SEQUENCER3_API_PORT:$ESPRESSO_SEQUENCER_API_PORT"
+    # Run the API server (with options taken from the environment)
+    command: sequencer -- http -- query
+    environment:
+      - ESPRESSO_SEQUENCER_ORCHESTRATOR_URL
+      - ESPRESSO_SEQUENCER_DA_SERVER_URL
+      - ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL
+      - ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_SEQUENCER_STORAGE_PATH
+      - ESPRESSO_SEQUENCER_L1_PROVIDER
+      - ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG
+      - RUST_LOG
+      - RUST_LOG_FORMAT
+    depends_on:
+      orchestrator:
+        condition: service_healthy
+    stop_grace_period: 1s
+
+  sequencer4:
+    image: ghcr.io/espressosystems/espresso-sequencer/sequencer:main
+    ports:
+      - "$ESPRESSO_SEQUENCER4_API_PORT:$ESPRESSO_SEQUENCER_API_PORT"
+    # Run the API server (with options taken from the environment)
+    command: sequencer -- http
+    environment:
+      - ESPRESSO_SEQUENCER_ORCHESTRATOR_URL
+      - ESPRESSO_SEQUENCER_DA_SERVER_URL
+      - ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL
+      - ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_SEQUENCER_STORAGE_PATH
+      - ESPRESSO_SEQUENCER_L1_PROVIDER
+      - ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG
+      - RUST_LOG
+      - RUST_LOG_FORMAT
+    depends_on:
+      orchestrator:
+        condition: service_healthy
+    stop_grace_period: 1s
+
+  commitment-task:
+    image: ghcr.io/espressosystems/espresso-sequencer/commitment-task:main
+    ports:
+      - "$ESPRESSO_COMMITMENT_TASK_PORT:$ESPRESSO_COMMITMENT_TASK_PORT"
+    command: commitment-task --deploy
+    environment:
+      - ESPRESSO_SEQUENCER_ETH_MNEMONIC
+      - ESPRESSO_SEQUENCER_HOTSHOT_ACCOUNT_INDEX
+      - ESPRESSO_COMMITMENT_TASK_PORT
+      - ESPRESSO_SEQUENCER_URL
+      - ESPRESSO_SEQUENCER_L1_PROVIDER
+      - ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG
+      - RUST_LOG
+      - RUST_LOG_FORMAT
+    depends_on:
+      sequencer0:
+        condition: service_healthy
+      demo-l1-network:
+        condition: service_healthy
+    stop_grace_period: 1s
+
+  example-rollup:
```

Comment:
This is the only service that should be kept. The others can be removed. Instead the documentation can mention to run `just demo` from the `espresso-sequencer` checkout and after that has started up `just demo` in this repo to deploy to existing sequencer network. Once everything is working let's also add the example rollup to the parent demo repo: https://github.com/EspressoSystems/espresso-rollup-demo